### ISSUE TITLE
Room for installation documentation improvement?

### ISSUE THREAD

**Issue by dsvensson** [helm/helm#4054]
I had to run the following commands to get tiller working on my Raspberry Pi.

First to get the tiller pod running at all:
```
kubectl taint nodes --all node-role.kubernetes.io/master-
```

Then a service account, which is slightly mentioned in the documentation, but could perhaps be explained in more detail?
```
kubectl create serviceaccount --namespace kube-system tiller
  serviceaccount "tiller" created

kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
  clusterrolebinding "tiller-cluster-rule" created

kubectl patch deploy --namespace kube-system tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}'
  deployment "tiller-deploy" patched
```

Output of `helm version`:
```
Client: &version.Version{SemVer:"v2.8+unreleased", GitCommit:"5f4b260ae50544c6565f68db7079d8ce55c35b6e", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.8+unreleased", GitCommit:"5f4b260ae50544c6565f68db7079d8ce55c35b6e", GitTreeState:"clean"}
```

Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.2", GitCommit:"81753b10df112992bf51bbc2c2f85208aad78335", GitTreeState:"clean", BuildDate:"2018-04-27T09:22:21Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/arm"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.2", GitCommit:"81753b10df112992bf51bbc2c2f85208aad78335", GitTreeState:"clean", BuildDate:"2018-04-27T09:10:24Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/arm"}
```

Helm/Tiller built from source, as no binaries available. Kubernetes from official apt repository.

**Comment by dsvensson** [comment]
Yep. Not sure what level the documentation should be wrt knowing about single k8s scheduling default settings. I'm very new to this, so it took a while to figure out the taint part. Now that I rechecked the documentation, what I did wrt tiller role ought to be equivalent to what is described here:

https://docs.helm.sh/using_helm/#role-based-access-control

..which I did read, but obviously gave too little attention to and instead jumped the google train and ended up filing this issue.
